app/vmctl/influx/influx.go

```diff
@@ -86,10 +88,10 @@ type LabelPair struct {
 // configured with passed Config
 func NewClient(cfg Config) (*Client, error) {
 	c := influx.HTTPConfig{
-		Addr:               cfg.Addr,
-		Username:           cfg.Username,
-		Password:           cfg.Password,
-		InsecureSkipVerify: true,
```

Comment:
Hi @khushijain21 ! Let's leave the `InsecureSkipVerify` feature because not all users want to set up TLS for the migration process. But we can add TLSConfig if it is needed